### ISSUE TITLE
Error on Install

### ISSUE THREAD

**Issue by ahaws-govcio** [formio/react-app-starterkit#65]
I am exploring that capabilities of FormIO and looking at using it in our React application, however, cloning the repo and executing `npm install` doesn't work.
I'll continue to work through the issue, but I would think this is something the starter application repo should resolve.

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: react-starterkit-app@0.1.0
npm ERR! Found: react@18.2.0
npm ERR! node_modules/react
npm ERR!   react@"^18.1.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^17.0.0 || ^16.3.0 || ^15.5.4" from react-json-view@1.21.3
npm ERR! node_modules/react-json-view
npm ERR!   react-json-view@"^1.21.3" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
```

**Comment by brendanbond** [comment]
did you try with the --legacy-peer-deps flag? FWIW we use yarn internally, it may be we just need to migrate this repo

**Comment by ahaws-govcio** [comment]
@brendanbond , yes, I did continue working through the issue and used the --legacy-peer-deps and was able to get the install to execute. I raised the issue, because that wasn't documented and figured it should be resolved in the repo or at least in the documentation. FWIW.

**Comment by brendanbond** [comment]
Thanks @ahaws-govcio , and understood. There has recently been a big internal push to get the React and Angular libraries up to date and modernized, but unfortunately these "starter-kit" type apps will probably be downstream of those changes as we focus on making sure the underlying libraries themselves are ready for prime time. Stay tuned!

**Comment by ZenMasterJacob20011** [comment]
Thank you for taking the time to report this issue. It seems that the react-json-view has been deprecated and is no longer being updated. However there is a fork of the repository being maintained by other contributors. Ticket FIO-8363 has been created for this issue.
